modules/cnf-installing-the-performance-addon-operator.adoc

```diff
@@ -0,0 +1,150 @@
+//CNF-78
+// * networking/multiple_networks/configuring-sr-iov.adoc
+
+[id="installing-the-performance-addon-operator_{context}"]
+= Installing the Performance Addon Operator
+
+Performance Addon Operator provides the ability to enable advanced node performance tunings on a set of nodes.
+As a cluster administrator, you can install Performance Addon Operator using the {product-title} CLI or the web console.
+
+[id="install-operator-cli_{context}"]
+== Installing the operator using the CLI
+
+As a cluster administrator, you can install the operator using the CLI.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware.
+* The {product-title} Command-line Interface (CLI), commonly known as `oc`.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the Performance Addon Operator by completing the following actions:
+
+.. Create the following Namespace Custom Resource (CR) that defines the `openshift-performance-addon-operator` namespace,
+and then save the YAML in the `pao-namespace.yaml` file:
++
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: performance-addon-operator
+  labels:
+    openshift.io/run-level: "1"
+----
+
+.. Create the namespace by running the following command:
++
+----
+$ oc create -f pao-namespace.yaml
+----
+
+. Install the Performance Addon Operator in the namespace you created in the previous step by creating the following objects:
+
+.. Create the following OperatorGroup CR and save the YAML in the
+pao-operatorgroup.yaml` file:
++
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-performance-addon-operator
+spec:
+  targetNamespaces:
+  - openshift-performance-addon-operator
+----
+
+.. Create the OperatorGroup CR by running the following command:
++
+----
+$ oc create -f pao-operatorgroup.yaml
+----
+
+.. Run the following command to get the `channel` value required for the next
+step.
++
+----
+$ oc get packagemanifest performance-addon-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
+
+4.4
+----
+
+.. Create the following Subscription CR and save the YAML in the `pao-sub.yaml` file:
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: performance-addon-operator-subscription
+  namespace: openshift-performance-addon-operator
+spec:
+  channel: <channel> <1>
+  name: performance-addon-operator
+  source: redhat-operators <2>
+  sourceNamespace: openshift-marketplace
+----
+<1> Specify the value from you obtained in the previous step for the `.status.defaultChannel` parameter.
+<2> You must specify the `redhat-operators` value.
+
+.. Create the Subscription object by running the following command:
++
+----
+$ oc create -f pao-sub.yaml
+----
+
+.. Change to the `openshift-performance-addon-operator` project:
++
+----
+$ oc project openshift-performance-addon-operator
```

Comment:
We're now placing command output in separate code blocks from the command itself. New style change to help with copy and paste functionality.

Reply:
Removed the output.